src/oca_github_bot/config.py

```diff
@@ -47,6 +47,10 @@ def func_wrapper(*args, **kwargs):
 
 BROKER_URI = os.environ.get("BROKER_URI", os.environ.get("REDIS_URI", "redis://queue"))
 
+MAINTAINER_ROLE_PAGE = os.environ.get(
+    "MAINTAINER_ROLE_PAGE", "https://odoo-community.org/page/maintainer-role"
+)
```

Comment:
```suggestion
MAINTAINER_ROLE_PAGE = os.environ.get("MAINTAINER_ROLE_PAGE")
```

For compatibility with existing deployments.